### ISSUE TITLE
Relaxing `pydantic[dotenv]` bounds

### ISSUE THREAD

**Issue by zzstoatzz** [firebolt-db/firebolt-python-sdk#227]
Hello!

I'm writing integrations for [Prefect](https://github.com/PrefectHQ/prefect) on top of this sdk and running into issues with the enforced dependencies' versions.

Specifically, what I'm working on requires `pydantic >= 1.10` so I'm running into this when using `firebolt-sdk==0.13.1`
```
ERROR: Cannot install prefect, pydantic[dotenv]==1.8.2, pydantic[dotenv]==1.9.0, pydantic[dotenv]==1.9.1 and pydantic[dotenv]==1.9.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    prefect 2.7.0 depends on pydantic>=1.10.0
    fastapi 0.70.0 depends on pydantic!=1.7, !=1.7.1, !=1.7.2, !=1.7.3, !=1.8, !=1.8.1, <2.0.0 and >=1.6.2
    pydantic[dotenv] 1.9.2 depends on pydantic 1.9.2
```

and if I try to use an earlier version, e.g.`firebolt-sdk==0.8.0` I run into similar issues with `httpx[http2]`:
```
The conflict is caused by:
    prefect 2.7.0 depends on httpx[http2]>=0.23
    firebolt-sdk 0.8.0 depends on httpx[http2]==0.21.3
```

I'm wondering if it's possible for the upper bound of `pydantic[dotenv]` to be increased (or removed entirely) [here](https://github.com/firebolt-db/firebolt-python-sdk/blob/main/setup.cfg#L31) to allow `pydantic==1.10`, as `httpx[http2]==0.23.0` is no longer an issue (as of `firebolt-sdk==0.13.1`).

**Comment by stepansergeevitch** [comment]
Hello @zzstoatzz,
Thank you for your issue!
I've already relaxed the pydantic version in this PR: https://github.com/firebolt-db/firebolt-python-sdk/pull/232
This will be availble as soon as it will get merged and released

**Comment by zzstoatzz** [comment]
@stepansergeevitch 

great! thank you

**Comment by stepansergeevitch** [comment]
Released in 0.15.0
